perun-scim/src/main/java/cz/metacentrum/perun/scim/api/entities/GroupSCIM.java

```diff
@@ -0,0 +1,100 @@
+package cz.metacentrum.perun.scim.api.entities;
+
+
+import org.codehaus.jackson.annotate.JsonProperty;
+
+import java.util.List;
+
+import org.codehaus.jackson.map.annotate.JsonSerialize;
+
+/**
+ * Group resource type for SCIM protocol.
+ *
+ * @author Sona Mastrakova <sona.mastrakova@gmail.com>
+ * @date 08.10.2016
+ */
+
+@JsonSerialize(include = JsonSerialize.Inclusion.NON_NULL)
+public class GroupSCIM extends Resource {
```

Comment:
If it is possible, it would be better to use composition instead of inheritance. It may cause problems (e.g. implementation of equals and hashCode methods).